packages/editor/src/store/utils/get-template-part-icon.js

```diff
@@ -1,19 +1,14 @@
 /**
  * WordPress dependencies
  */
-import { layout, header, footer } from '@wordpress/icons';
-
+const icons = require( '@wordpress/icons' );
```

Comment:
```suggestion
import * as icons from '@wordpress/icons';
```

Reply:
Ah yes, thank you!  Thats what I was looking for. 😆  🤦‍♀️